src/components/metadata/metadata-row.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import MetaDataList from './metadata-list';
+import MetaDataValue from './metadata-value';
+import './styles/metadata.css';
+
+/**
+ * Shows metadata label and a given single value, or a list of values, or child elements
+ */
+const MetaDataRow = ({
+  label,
+  property,
+  kind = 'text',
+  empty = '-',
+  visible = true,
+  inline = true,
+  commas = true,
+  children
+}) => {
+  const showElements = React.isValidElement(children);
+  const showList = !showElements && Array.isArray(children) && children.length;
+  const showValue = !showList && !showElements;
```

Comment:
```suggestion
  const showValue = !showList;
```
`!showList` implies `!showElements`

Reply:
Yeah it is redundant but I left it in to make it easier to read, but it seems clear enough then so I'll remove it.